src/controllers/company.number.controller.ts

```diff
@@ -0,0 +1,57 @@
+import {NextFunction, Request, Response} from "express";
+import {check, validationResult} from "express-validator/check";
+import * as errorMessages from "../model/error.messages";
+import {createGovUkErrorData, GovUkErrorData} from "../model/govuk.error.data";
+import * as templatePaths from "../model/template.paths";
+import {ValidationError} from "../model/validation.error";
+
+// validator middleware
```

Comment:
'// validator middleware' here and on the function below isn't a very descriptive comment. Could you add some more detail, like you did for 'padCompanyNumber' below?